wai-extra/wai-extra.cabal

```diff
@@ -206,6 +208,7 @@ test-suite spec
                    , time
                    , case-insensitive
                    , http2
+                   , aeson
```

Comment:
This is already a dependency of the library, so this isn't adding a new dependency